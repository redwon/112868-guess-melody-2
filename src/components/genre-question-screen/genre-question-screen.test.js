import React from 'react';
import renderer from 'react-test-renderer';

import GenreQuestionScreen from './genre-question-screen';

const mockQuestion = {
  type: `genre`,
  genre: `genre`,
  answers: [
    {
      src: `src`,
      genre: `genre`,
    },
  ],
};

it(`renders correctly`, () => {
  const tree = renderer
    .create(<GenreQuestionScreen
      question={mockQuestion}
      screenIndex={0}
      onAnswer={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
