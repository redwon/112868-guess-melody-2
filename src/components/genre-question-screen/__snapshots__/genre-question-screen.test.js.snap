// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Выберите 
    genre
     треки
  </h2>
  <form
    className="game__tracks"
    onSubmit={[Function]}
  >
    <div
      className="track"
    >
      <button
        className="track__button track__button--play"
        onClick={[Function]}
        type="button"
      />
      <div
        className="track__status"
      >
        <audio
          src="src"
        />
      </div>
      <div
        className="game__answer"
      >
        <input
          className="game__input visually-hidden"
          id="answer-0"
          name="answer"
          onChange={[Function]}
          type="checkbox"
          value="answer-0"
        />
        <label
          className="game__check"
          htmlFor="answer-0"
        >
          Отметить
        </label>
      </div>
    </div>
    <button
      className="game__submit button"
      type="submit"
    >
      Ответить
    </button>
  </form>
</section>
`;
